src/api/youtube-api.js

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import { YOUTUBE_SEARCH_API_URL } from './endpoints';
+import { YOUTUBE_API_KEY } from './api-keys';
+
+export const fetchSearchResults = async query => {
+  try {
+    return await axios.get(YOUTUBE_SEARCH_API_URL, {
```

Comment:
store the result api in a variable and then return that variable to make your code more readable

```
const response = await axios.get(YOUTUBE_SEARCH_API_URL, {});
return response;
```